conf/test.config

```diff
@@ -34,4 +34,5 @@ params {
     hg38_ref_fai = "${projectDir}/test/reference/hg38.fa.fai"
     hg38_ref_dict = "${projectDir}/test/reference/hg38.dict"
```

Comment:
Paths are local and not particularly useful. Either remove or make small reference file